content/guides/production-build.md

```diff
@@ -71,8 +71,90 @@ module.exports = {
   /*...*/
 };
 ```
+When we do have multiple configurations in mind for different environments, the easiest way is to write seperate js files for 
+each environment. For example:
+##config/dev.js
+```js
+module.exports = function(env) {
+    debug: true,
+    devtool: 'cheap-module-source-map',
+    output: {
+      path: path.join(__dirname, '/../dist/assets'),
+      filename: '[name].bundle.js',
+      publicPath: publicPath,
+      sourceMapFilename: '[name].map'
+    },
+    devServer: {
+      port: 7777,
+      host: 'localhost',
+      historyApiFallback: true,
+      noInfo: false,
+      stats: 'minimal',
+      publicPath: publicPath
+    },
+ });
+```
+##config/prod.js
+```js
+module.exports = function(env) {
+debug: false,
+    output: {
+      path: path.join(__dirname, '/../dist/assets'),
+      filename: '[name].bundle.js',
+      publicPath: publicPath,
+      sourceMapFilename: '[name].map'
+    },
+    plugins: [
+      new webpack.LoaderOptionsPlugin({
+      minimize: true,
+      debug: false
+    }),
+      new UglifyJsPlugin({
+        beautify: false,
+        mangle: { screw_ie8 : true, keep_fnames: true },
+        compress: { screw_ie8: true },
+        comments: false
+      })
+    ]
+});
+```
+Have our webpack.config.js has the following snippet:
+```js
+function buildConfig(env) {
+  var config;
+  switch (env) {
```

Comment:
You are right. The switch-case is just a fail-safe, which I felt was necessary for my project, just in case. As for the example, am with your suggestion. It is much simpler. Thanks.